### ISSUE TITLE
[Request] Create Repository for .N3S Files.

### ISSUE THREAD

**Issue by frozenLake** [and0p/NesVoxel#9]
While N3S is not done yet, I can imagine that a few people have made some good .n3s files for their games, and these likely are worth sharing.

I mean, while one could make an unofficial repository for this stuff, it would be far more effective to have a singluar one for people to contribute to.

**Comment by and0p** [comment]
Agreed! But I don't want to put the carriage before the horse. Between outlines and dynamic meshes, which are almost fully implemented, there are still a few new features in terms of how you're able to model the game. (It's all backwards compatible though.) I wanna get all that set up first.

It's definitely important to have a cental repo. Automatic downloads would be ideal. That being said, there are a few obstacles:

- A centralized repo would be expensive, since N3S files, even zipped, are kinda big.
- A centralized repo would either be curated or require an account system / ability to upload and vote. That's not easy to make, at least when you have to guard it from abuse.
- Automatic downloading requires standardized ROM naming / identification.
- Automatic downloading requires naming one as an "official" N3S definition, which will require active curation.

Right now the N3S site is a wiki. I'm planning on making a big game list, and letting people add their own N3S files, screenshots, and notes. I'm also thinking Github would be great for revisions, collaborations, and downloads ("releases"), so that will hopefully be the file host.
